src/components/highlight-component/highlight-component.js

```diff
@@ -49,9 +64,9 @@ function HighlightComponent() {
 
 HighlightComponent.prototype.toggleComponents = function () {
   this.state = !this.state;
-
+  // TODO: Reformat this loop to make it more readable. 
```

Comment:
This loop feels okay in terms of readability? You could maybe remove the comment.

Reply:
Ahhh yes, I had quite a number of these TODO comments and missed this one out. I'll remove it :)